optimizer_wrapper.rb

```diff
@@ -591,7 +590,7 @@ def self.apply_zones(vrp)
   end
 
   def self.clique_cluster(vrp, cluster_threshold, force_cluster)
-    @zip_condition = cluster_threshold && vrp.matrices.any? && vrp.services.all?(&:activity) &&
+    @zip_condition = cluster_threshold && vrp.matrices.any? && vrp.services.all?(&:activity) && vrp.rests.empty? &&
```

Comment:
I just realise this condition. Why do we want to skip zip_cluster because of a vehicle with pause?

Reply:
Zip cluster ignore the timewindows. The upper level route is build with compressed zip_clusters. So when the solution is then expended with unzip_cluster and the result of the TSPs the times are shifted. We cannot ensure that the rests stays feasible

Reply:
Cheers for the explanation